### ISSUE TITLE
Update the DOI in the A single-cell transcriptional roadmap of the mouse and human lymph node lymphatic vasculature collection

### ISSUE THREAD

**Issue by brianraymor** [chanzuckerberg/single-cell-curation#106]
The `DOI` in [A single-cell transcriptional roadmap of the mouse and human lymph node lymphatic vasculature](https://cellxgene.cziscience.com/collections/9c8808ce-1138-4dbe-818c-171cff10e650) references a [preprint](https://www.biorxiv.org/content/10.1101/2019.12.31.892166v1) which includes a note:

> Now published in Frontiers in Cardiovascular Medicine doi: 10.3389/fcvm.2020.00052

**Comment by brianraymor** [comment]
And crossref returns:
```

    "relation": {
      "is-preprint-of": [
        {
          "id-type": "doi",
          "id": "10.3389/fcvm.2020.00052",
          "asserted-by": "subject"
        }
      ]
    },
```

**Comment by jahilton** [comment]
addressed in #144
